.neutrinorc.js

```diff
@@ -1,65 +1,81 @@
 module.exports = {
   options: {
-    root: __dirname
+    root: __dirname,
   },
   use: [
-    ['./packages/airbnb', {
-      // See the package.json `lint` script for which files are linted.
-      // Excludes are managed via `.eslintignore`.
-      eslint: {
-        baseConfig: {
-          extends: ['prettier'],
-          env: {
-            browser: true,
-            jest: true,
-            mocha: true,
-            node: true
-          },
-          plugins: ['prettier'],
-          rules: {
-            // Disallow trailing commas on arrays, objects, functions, et al
-            'comma-dangle': ['error', 'never'],
-            // Allow using console since most of the code in this repo isn't run in a browser.
-            'no-console': 'off',
-            // Allowing shadowing variable that share the same context as the outer scope
-            'no-shadow': 'off'
-          },
-          overrides: [
-            {
-              files: ['packages/create-project/commands/init/templates/**'],
-              rules: {
-                // The dependencies in create-project's templates are installed by
-                // by create-project and so are expected to be missing from package.json.
-                'import/no-extraneous-dependencies': 'off'
-              }
+    [
+      './packages/airbnb',
+      {
+        // See the package.json `lint` script for which files are linted.
+        // Excludes are managed via `.eslintignore`.
+        eslint: {
+          baseConfig: {
+            extends: ['plugin:prettier/recommended'],
+            env: {
+              browser: true,
+              jest: true,
+              mocha: true,
+              node: true,
             },
-            {
-              files: ['packages/create-project/commands/init/templates/preact/**'],
-              settings: {
-                react: {
-                  pragma: 'h'
-                }
-              },
-              rules: {
-                // With Preact the use of `class` is recommended over `className`,
-                // so we have to add `class` to the ignore list, to prevent:
-                // `Unknown property 'class' found, use 'className' instead`
-                'react/no-unknown-property': ['error', { ignore: ['class'] }]
-              }
+            rules: {
+              'prettier/prettier': [
```

Comment:
Do most editors support reading the Prettier config out of the `.eslintrc.js` file, or should we move the prettier config to `.prettierrc.js` (which `eslint-plugin-prettier` reads too)? eg:
https://github.com/mozilla/treeherder/blob/master/.prettierrc.js

Reply:
I've used both, and they each seem to work as expected. We can certainly add a command for it.